torch/_refs/__init__.py

```diff
@@ -1414,18 +1414,23 @@ def std(
     if dim == () or dim == []:
         dim = None
 
+    opmath_dtype = utils.get_computation_dtype(a.dtype)
+    dtype = a.dtype
+    if utils.is_complex_dtype(dtype):
+        dtype = utils.corresponding_real_dtype(dtype)
```

Comment:
```suggestion
    opmath_dtype, dtype = utils.reduction_dtypes(a, REDUCTION_OUTPUT_TYPE_KIND.COMPLEX_TO_FLOAT)
```

Reply:
I noticed that `std` is truncated before taking sqrt after I landed it, but decided that who cares ;-)

Reply:
test_decomp cares lol